### PR TITLE
zutty: update to 0.12

### PR DESCRIPTION
**Pull request by nadevko**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, _X86_64_
- I built this PR locally for these architectures _aarch64-musl_

### CODE REVIEW

**Comment by nadevko** [comment]
That's strange, but at first time it successfully compiled with wrong checksum.

File: srcpkgs/zutty/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-license="GPL-3.0-or-later"
-homepage="https://tomscii.sig7.se/zutty"
+license=GPL-3.0-or-later
+homepage=https://tomscii.sig7.se/zutty
 distfiles="https://github.com/tomszilagyi/zutty/archive/refs/tags/${version}.tar.gz"
-checksum=3e77abb424f2f6044686fd3a3a22793e537bf5e403fe2b6ca8543daf4f3186f9
+checksum=750daed8625a2486a33b872d59242efba7ee21af282ccfd4a7dbf12d21d7cee7
 
 post_patch() {
 	vsed -i wscript -e "s|, '-march=native', '-mtune=native'||g"
```
Comment (paper42) [inline srcpkgs/zutty/template:17]: if you look at the CI output, you will see that this line doesn't change anything and causes the build to fail

**Comment by nadevko** [comment]
`python3-devel` are not really required, from `MesaLibs` required only `libglvnd-devel`, but package `font-misc-misc` provides default fonts.
<sup><sup><sub>When I learn to push only final version...</sub></sup></sup>

**Comment by paxet** [comment]
Only by few hours of difference we have duplicated our effort #37252

**Comment by paper42** [comment]
I will merge @paxet's PR just because the diff is a bit cleaner, but thank you for looking into this and testing.

**Pull request closed without merging**